FreeRTOS_IP.c

```diff
@@ -1009,6 +1025,18 @@ static void prvCheckNetworkTimers( void )
         ( void ) xSendEventToIPTask( eARPTimerEvent );
     }
 
+    /* Is the ARP resolution timer expired? */
+    if( prvIPTimerCheck( &xARPResolutionTimer ) != pdFALSE )
+    {
+        if( pxARPWaitingNetworkBuffer != NULL )
```

Comment:
These timers are auto-repeating. Let's switch it off when it expires:
~~~c
     /* Is the ARP resolution timer expired? */
     if( prvIPTimerCheck( &xARPResolutionTimer ) != pdFALSE )
     {
+        vIPSetARPResolutionTimerEnableState( pdFALSE );
+
         if( pxARPWaitingNetworkBuffer != NULL )
~~~

Reply:
Yes, done. Thank you.